charts/karpenter-crd/values.yaml

```diff
@@ -0,0 +1,7 @@
+webhook:
+  # -- Whether to enable the webhooks and webhook permissions.
+  enabled: true
+  serviceName: karpenter
+  serviceNamespace: kube-system
```

Comment:
The idea here is that users may want to add their crd chart in a different name or change the service name to a different name

Reply:
This was my callout last week. I think we closed on the fact that we were going to give the flexibility for users to install these in different namespaces

Reply:
Gotcha. Thinking about it more, is it something that people do to have their installation in a different namespace than the namespace they install the resources in? Since the namespace just modifies the namespace of the service that we're referencing, and nothing else, I'd say that we should just use the name of the release. Am I misunderstanding?

Reply:
> Thinking about it more, is it something that people do to have their installation in a different namespace than the namespace they install the resources in? 

Do you see a problem with giving customers this flexibility?

Reply:
Synced offline, good to keep as is.